fiftyone/utils/transformers.py

```
@@ -1510,7 +1519,9 @@ def __call__(
 
         return res
 
-    def _parse_output(self, output, frame_size, confidence_thresh, classes=None):
+    def _parse_output(
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_parse_output`

(ANN202)